plc4j/drivers/opcua/src/main/java/org/apache/plc4x/java/opcua/OpcuaPlcDriver.java

```diff
@@ -36,16 +35,15 @@ Licensed to the Apache Software Foundation (ASF) under one
  * Implementation of the OPC UA protocol, based on:
  * - Eclipse Milo (https://github.com/eclipse/milo)
  *
- * @author Matthias Milan Stlrljic
- * Created by Matthias Milan Stlrljic on 10.05.2019
+ * @author Matthias Milan Strljic
```

Comment:
Its recommended to not use @author tag on Apache Licensed files for multiple reasons, I suggest to remove it thus.

Reply:
Should be fixed now my lord